src/initializers/newrelic/index.ts

```diff
@@ -1,9 +1,12 @@
-import {  OrkaOptions } from '../../typings/orka';
+import { OrkaOptions } from '../../typings/orka';
 
+let newrelic;
 export default async (config, orkaOptions: Partial<OrkaOptions>) => {
   if (config.newRelicLicenseKey) {
     process.env.NEW_RELIC_HOME = __dirname;
     process.env.NEW_RELIC_APP_NAME = `${orkaOptions.appName} ${config.nodeEnv}`;
-    await import('newrelic');
+    newrelic = await import('newrelic');
   }
 };
+
+export { newrelic };
```

Comment:
does this work? Have you tested it?

Reply:
Tested with console logs and it gets updated.